xml/geo_manage_i.xml

```diff
@@ -164,25 +197,115 @@ booth[27900]: 2014/08/13_10:21:23 info: revoke succeeded!</screen>
+    </para>
+<screen>&prompt.root;<command>booth</command> grant -a 192.168.202.100 &ticket1;</screen>
+     <para>Resources depending on &ticket1; will automatically fail over to
+       <literal>&cluster1;</literal> in case
+       <literal>&cluster2;</literal> fails.</para>
+   </step>
   </procedure>
-  <para>
-   <remark>taroth 2018-04-03: FATE#322100 - add how to proceed for manual failover
-    of tickets</remark>
-  </para>
- </sect1>
+ </sect2>
+ <sect2 xml:id="sec.ha.geo.manage.cli.booth.man-ticket.move">
+  <title>Moving a Manual Ticket</title>
+   <para>
+    Assuming that you want to manually move <literal>ticketnfs</literal> from
+    site <literal>&cluster1;</literal> (with the virtual IP
+    <literal>192.168.201.100</literal>) to site <literal>&cluster2;</literal>
+    (with the virtual IP <literal>192.168.202.100</literal>), proceed as
+    follows:
+   </para>
+  <procedure>
+   <step>
+    <para>Log in to <literal>&cluster1;</literal>.</para>
+   </step>
+   <step>
+    <para>
+     Set <literal>&ticket3;</literal> to standby with the following command:
+    </para>
+<screen>&prompt.root;<command>crm_ticket</command> -t &ticket3; -s</screen>
+   </step>
+   <step>
+    <para>
+     Wait for any resources that depend on <literal>&ticket3;</literal> to be
+     stopped or demoted cleanly.
+    </para>
+   </step>
+   <step>
+    <para>
+     Grant <literal>&ticket3;</literal>to the site <literal>&cluster2;</literal> with:
+    </para>
+<screen>&prompt.root;<command>booth</command> grant -s 192.168.202.100 -F &ticket3;</screen>
+   </step>
+   <step>
+    <para>
+     Activate <literal>&ticket3;</literal> on site <literal>&cluster1;</literal> with:
+    </para>
+<screen>&prompt.root;<command>booth</command> grant -a 192.168.202.100 &ticket1;</screen>
+     <para>Resources depending on <literal>&ticket3;</literal> will automatically fail over to
+       <literal>&cluster1;</literal> in case
+       <literal>&cluster2;</literal> fails.</para>
+   </step>
+  </procedure>
+ </sect2>
+
+ <sect2 xml:id="sec.ha.geo.manage.cli.booth.man-ticket.failover">
+  <title>Failing Over a Manual Ticket</title>
+   <para> Let us assume that the (manually managed) ticket
+     <literal>&ticket3;</literal> had been granted to site
+     <literal>&cluster1;</literal> (with the virtual IP
+     <literal>192.168.201.100</literal>. This site cannot be reached at the
+    moment. Site <literal>&cluster2;</literal> (with the virtual IP
+     <literal>192.168.202.100</literal>) is still available. </para>
+   <para> If you manage to contact a local administrator on
+     <literal>&cluster1;</literal>, you can verify if the site is really down.
+    In that case, the ticket on <literal>&cluster1;</literal> cannot be revoked
+    and the resources depending on this ticket cannot be started on
+     <literal>&cluster2;</literal> unless you manually grant the ticket
+     <literal>&ticket3;</literal> to <literal>&cluster2;</literal>. To do so,
```

Comment:
The comment applies in here now ;-)
```
ygao 2018-06-26: @taroth:How about to rephrase "In that case ...to cluster2" to be something like:
"Make sure the ticket, or basically the relevant resources, have been relinquished
from cluster1. If we are sure now it's safe and ready to grant the ticket to cluster2
and host the relevant resources there, proceed as follows:"
```

Reply:
Cluster1 cannot be reached for a reason. It might be really down, which is not bad :-) -- We can do the move right away. It might not be really down though, for example if there's network connectivity issue. If it's latter, and if we want to switch things to cluster2, in order to prevent any concurrency violations, we should still ask admin at cluster1 to revoke/standby the ticket over there by itself,  so that the cluster/site basically relinquishes the resources,